include/swift/IDE/RefactoringKinds.def

```diff
@@ -54,6 +54,12 @@ CURSOR_REFACTORING(MemberwiseInitLocalRefactoring, "Generate Memberwise Initiali
 
 CURSOR_REFACTORING(AddEquatableConformance, "Add Equatable Conformance", add.equatable.conformance)
 
+CURSOR_REFACTORING(ConvertCallToAsyncAlternative, "Convert Call to Async Alternative", convert.call.to.async)
```

Comment:
I always thought the dot separation here was to establish some form of grouping/hierarchy across refactorings like it is for rename and extract, but I guess it became a word separator at some point.

Reply:
That would make sense. I'm happy to change the added ones, eg. "convert.range.call-to-async", "convert.func.to-async", and "add.func.async-alternative". I assume changing the others would be a Bad Idea (TM) though.

(FWIW, the grouping I'm imagining is eg. "convert", "extract", and "add" as the top level, with "range" or "func" as the next. Some examples there would be "add.range.expand-default", "add.range.expand-switch-cases", "convert.range.localize-string")

Reply:
Yeah they're meant to stay stable once introduced, so too late now. I don't mind either way for the new ones. This was more just me musing out loud in a comment than making a suggestion.

Reply:
Ended up doing `convert.call-to-async`, etc. since the range/function didn't add much.